Dockerfile

```diff
@@ -1,5 +1,8 @@
+# Adding arguments
+ARG TAG=latest
```

Comment:
Perhaps `dev` or `local`? I don't think we want our local tag to overwrite our prod tag.

Alternatively the image built off main could be `prod` or `main` rather than `latest` if you want to keep `latest` here?

Reply:
Good catch!